src/main/java/shortestpath/Transport.java

```diff
@@ -187,7 +201,7 @@ public class Transport {
         }
 
         if (fieldMap.containsKey("Varbits")) {
-            for (String varbitCheck : fieldMap.get("Varbits").split(DELIM)) {
+            for (String varbitCheck : fieldMap.get("Varbits").split(";")) {
```

Comment:
No exist row from a tsv used multipe varbits so the delimiter didnt matter before. I just though `;` was easier to read than a space ` ` for the new spell_transports.tsv